src/sectionTitle.story.jsx

```diff
@@ -0,0 +1,41 @@
+
+import React from 'react';
+import Bounds from './Bounds';
+import Section from './Section';
+import SectionTitle from './SectionTitle';
+import { storiesOf } from '@kadira/storybook';
+
+storiesOf('SectionTitle', module)
+	.add('Default', () => (
```

Comment:
let's make sure to add the auto documentation, https://github.com/meetup/meetup-web-components/blob/master/src/bounds.story.jsx#L7